config/schedule.rb

```diff
@@ -24,3 +24,7 @@ def local(time)
 every :month, at: local('02:00 am'), roles: [:seed] do
   rake 'generate:seed[1]'
 end
+
+every :day, at: local('03:00 am'), roles: [:cron] do
+  rake 'refresh_materialized_db_views'
```

Comment:
Do we need to refresh the tables to get updated schemas, or to even get updated rows?

Reply:
We need to refresh the views to get updated rows. Since these are materialized, they only update when refreshed.